platform/viewer/public/config/default.js

```diff
@@ -24,6 +24,8 @@ window.config = {
     prefetch: 25,
   },
   // filterQueryParam: false,
+  // enable imageLoaderAcceptHeader configuration to force a reconfiguration of WADOImageLoader to support preflight request
```

Comment:
Do we still want these commented lines included?

Reply:
nope